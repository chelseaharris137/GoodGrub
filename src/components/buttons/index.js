export * from './Button';
export * from './LinkedButton';
