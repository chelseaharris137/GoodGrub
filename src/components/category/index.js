export * from './Categories';
export * from './CategoryItem';
