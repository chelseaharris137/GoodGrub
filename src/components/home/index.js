export * from './About';
export * from './HeroSection';
