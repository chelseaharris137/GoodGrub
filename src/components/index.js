export * from './buttons/Button';
export * from './buttons/LinkedButton';
export * from './category/Categories';
export * from './category/CategoryItem';
export * from './home/About';
export * from './home/HeroSection';
export * from './layout/Footer';
export * from './layout/Layout';
export * from './layout/Navbar';
export * from './mealsPage/IngredientsTable';
export * from './mealsPage/SearchBar';
export * from './mealsPage/SingleMealCard';
export * from './text/PointerText';
export * from './text/Text';
export * from './text/Title';
