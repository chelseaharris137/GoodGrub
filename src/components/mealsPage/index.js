export * from './IngredientsTable';
export * from './SearchBar';
export * from './SingleMealCard';
