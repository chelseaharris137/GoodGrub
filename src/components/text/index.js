export * from './PointerText';
export * from './Text';
export * from './Title';
