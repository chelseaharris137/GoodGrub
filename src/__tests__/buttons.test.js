import React from 'react';
import { render } from '@testing-library/react';
import { Button, LinkedButton } from '@/components';

describe('`Button` component', () => {
  it('Does `Button` match DOM Snapshot', () => {
    const button = render(<Button />);

    expect(button).toMatchSnapshot();
  });
});

describe('`LinkedButton` component', () => {
  it('Does `LinkedButton` match DOM Snapshot', () => {
    const linkedButton = render(<LinkedButton />);

    expect(linkedButton).toMatchSnapshot();
  });
});
