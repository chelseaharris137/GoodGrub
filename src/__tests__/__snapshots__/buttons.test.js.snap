// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Button\` component Does \`Button\` match DOM Snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <button
        class=""
        type="button"
      />
    </div>
  </body>,
  "container": <div>
    <button
      class=""
      type="button"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`\`LinkedButton\` component Does \`LinkedButton\` match DOM Snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <a
        class="undefined undefined"
        href="/"
        type="button"
      />
    </div>
  </body>,
  "container": <div>
    <a
      class="undefined undefined"
      href="/"
      type="button"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
